### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Limezy**

### CODE REVIEW

**Comment by Limezy** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -102,8 +102,8 @@ popd
 ynh_script_progression --message="Configuring PHP-FPM..." --weight=1
 
 # Create a dedicated PHP-FPM config
-ynh_add_fpm_config --usage="low" --footprint="low"
 phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
+ynh_add_fpm_config --usage="low" --footprint="low"
```
Comment (ericgaspar) [inline scripts/install:106]: you should keep the order `ynh_add_fpm_config` and the store `phpversion` (for install and upgrade)

Comment (Limezy) [reply scripts/install:106]: Ok but then it throws an error saying it doesn't know phpversion ?

**Comment by Limezy** [comment]
@ericgaspar my initial intention was to change the session time. Unfortunately I was still disconnected so apparently it doesn't work. Do you have any idea why ?

File: scripts/upgrade [hunk 2]
```diff
@@ -74,6 +74,10 @@ ynh_system_user_create --username=$app --home_dir="$final_path"
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 
+# Void and recreate final_path
+ynh_secure_remove --file="$final_path"
+ynh_exec_warn_less mkdir "$final_path"
```
Comment (ericgaspar) [inline scripts/upgrade:79]: Why is this added?

Comment (Limezy) [reply scripts/upgrade:79]: I don't remember exactly, but I think that without the source weren't well updated (old files kept?).

Comment (Limezy) [reply scripts/upgrade:79]: Ohh! @ericgaspar I just found that there was a `--full_replace` option for ynh_setup_source. Great !

File: scripts/install (hunk 1 above)
Comment (Limezy) [reply scripts/install:105]: I have tried to change this, could you please have a look @ericgaspar ?

File: scripts/upgrade (hunk 2 above)
Comment (Limezy) [reply scripts/upgrade]: Should be resolved now, but I have the feeling that unlike what is said in documentation, --full_replace doesn't work. I need to use --full_replace=1 instead

**Review by Limezy — COMMENTED** [review]
Do you think this should work @ericgaspar ?

**Comment by Limezy** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]
